acvm/src/pwg.rs

```diff
@@ -36,7 +40,11 @@ pub enum PartialWitnessGeneratorStatus {
     ///
     /// The caller must resolve these opcodes externally and insert the results into the intermediate witness.
     /// Once this is done, the `PartialWitnessGenerator` can be restarted to solve the remaining opcodes.
-    RequiresOracleData { required_oracle_data: Vec<OracleData>, unsolved_opcodes: Vec<Opcode> },
+    RequiresOracleData {
+        required_oracle_data: Vec<OracleData>,
+        unsolved_opcodes: Vec<Opcode>,
+        unresolved_brilligs: Vec<UnresolvedBrillig>,
```

Comment:
Sidenote but I quite dislike the use of the term Brillig as a unit. Brillig bytecode, Brillig VM, etc. are all good but "I need to resolve 3 Brilligs" is just meaningless in my mind.

Could we rename this to be a `BrilligCall` or something which gives some intuition about what it does?